homeassistant/components/idasen_desk/__init__.py

```diff
@@ -35,7 +28,7 @@ class DeskData:
     """Data for the Idasen Desk integration."""
 
     address: str
```

Comment:
Also wondering, is the address reachable from the coordinator? In that case we can just store the coordinator in the runtime data

Reply:
good point. dropped `DeskData` and used the Coordinator in `runtime_data` now